com.unity.ml-agents.extensions/Tests/Editor/Match3/Match3SensorTests.cs

```diff
@@ -18,7 +18,7 @@ public class Match3SensorTests
         public void TestVectorObservations()
         {
             var boardString =
-                @"000
+@"000
```

Comment:
intended?
(There's a few more below)

Reply:
No, coding package reformatted them (and a few other files). Will revert.